src/components/Person.jsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import './Person.scss';
+
+export function Person({ name, age, sex, isMarried, partnerName }) {
```

Comment:
For convenience, you can do so:
```suggestion
export function Person({
  name,
  age,
  sex,
  isMarried,
  partnerName
}) {
```